library/core/src/macros/mod.rs

```diff
@@ -795,7 +795,7 @@ pub(crate) mod builtin {
     ///
     /// Two such examples are macros and `#[cfg]` environments.
     ///
-    /// Emit better compiler error if a macro is passed invalid values. Without the final branch,
+    /// Emit better compiler errors if a macro is passed invalid values. Without the final branch,
```

Comment:
Not to be too pedantic, but I think there's only one error being emitted?  So perhaps "*Emit **a** better compiler error...*" would be better?